example/app/(examples)/DataBinding.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import {
+  SafeAreaView,
+  ScrollView,
+  StyleSheet,
+  Button,
+  View,
+} from 'react-native';
+import Rive, { Fit, RiveRef } from 'rive-react-native';
+
+export default function DataBinding() {
+  const riveRef = React.useRef<RiveRef>(null);
+
+  const updateDataBindingValues = () => {
+    // Update the view model instance properties
+    // NUMBER VALUE
+    riveRef.current?.setNumberPropertyValue('Energy_Bar/Lives', 7);
+    // STRING VALUE
+    riveRef.current?.setStringPropertyValue('Button/State_1', "Let's go!");
```

Comment:
I'm not seeing any text get updated.

Reply:
It's the text that is visible before pressing the button